buildbuddy.yaml

```diff
@@ -30,6 +32,7 @@ actions:
       pull_request:
         branches:
           - "*"
+        force_manual_merge_with_base: false
```

Comment:
while you're here do you mind also setting `git_fetch_depth: 0` on the action?

More explanation in the thread starting [here](https://github.com/buildbuddy-io/buildbuddy-internal/issues/3482#issuecomment-2151017215) but I think we cannot determine changed files unless either (a) we do a non-shallow fetch, or (b) if GitHub's generated merge commits are generated using `--squash`. (I'm not sure whether that's the case yet)

Reply:
err, actually it looks like GitHub's generated merge commit is always a merge commit even if commit history is linear. So we can set `git_fetch_depth: 2` instead of `git_fetch_depth: 0`, and we should be able to use `HEAD^1` as the diff base.